### ISSUE TITLE
[BUG] `transition-colors` and `transition-transform` get flagged as applying the same properties

### ISSUE THREAD

**Issue by nikitarevenco** [francoismassart/eslint-plugin-tailwindcss#364]
**Describe the bug**
`transition-colors` and `transition-transform` get flagged as applying the same properties

**To Reproduce**
this code:

```jsx
      <div className="transition-colors transition-transform" />
```

**Expected behavior**
The rule should not trigger

**Environment (please complete the following information):**
 - OS: Linux
 - Version used: 0.6.8

**Additional context**
transition-transform:

```css
.transition-transform {
  transition-property: transform;
  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
  transition-duration: 150ms;
}
```
transition-colors:

```css
.transition-colors {
  transition-property: color, background-color, border-color, text-decoration-color, fill, stroke;
  transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
  transition-duration: 150ms;
}
```

So maybe it becomes unhappy because of `transition-timing-function` and `transition-duration` being applied at once

**Comment by nikitarevenco** [comment]
Temporary solution (without disabling rule):

```css
      <div className="transition-colors [transition-property:transform]" />
```

**Comment by kachkaev** [comment]
I think that the rule is triggering correctly. It matches the list of property _names_ rather than names and values. If we set, say:

```css
.foo { background: red }
.foo { background: green }
```

we won’t get both red _and_ green for the background. Only one of the values will ‘win’. The same applies to `transition-property`.

**Comment by francoismassart** [comment]
Hi,
If you test your markup in the playground, you can see the official plugin itself consider that is a mistake...

https://play.tailwindcss.com/yCLGcdoMpG

<img width="953" height="320" alt="Image" src="https://github.com/user-attachments/assets/01dcaa2e-eb78-4a32-adda-3f1bf8bda5e6" />
